specification/azurestack/resource-manager/Microsoft.AzureStack/stable/2022-06-01/AzureStack.json

```diff
@@ -0,0 +1,504 @@
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/providers/Microsoft.AzureStack/cloudManifestFiles": {
+      "get": {
+        "x-ms-examples": {
+          "Returns the properties of a cloud specific manifest file with latest version.": {
+            "$ref": "examples/CloudManifestFile/List.json"
+          }
+        },
+        "tags": [
+          "AzureStack"
+        ],
+        "description": "Returns a cloud specific manifest JSON file with latest version.",
+        "operationId": "CloudManifestFile_List",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/CloudManifestFileResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.AzureStack/cloudManifestFiles/{verificationVersion}": {
+      "get": {
+        "x-ms-examples": {
+          "Returns the properties of a cloud specific manifest file.": {
+            "$ref": "examples/CloudManifestFile/Get.json"
+          }
+        },
+        "tags": [
+          "AzureStack"
+        ],
+        "description": "Returns a cloud specific manifest JSON file.",
+        "operationId": "CloudManifestFile_Get",
+        "parameters": [
+          {
+            "name": "verificationVersion",
+            "in": "path",
+            "description": "Signing verification key version.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "versionCreationDate",
+            "in": "query",
+            "description": "Signing verification key version creation date.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/CloudManifestFileResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.AzureStack/generateDeploymentLicense": {
+      "post": {
+        "x-ms-examples": {
+          "Creates a license that can be used to deploy an Azure Stack device.": {
+            "$ref": "examples/DeploymentLicense/Post.json"
+          }
+        },
+        "tags": [
```

Comment:
Whats the reason why this is a POST and not a PUT? We recommend using PUTs for creating resources that way you also get the benefit of using template deployments.